src/Sylius/Bundle/AdminBundle/Resources/config/routing/partial/taxon.yml

```diff
@@ -4,6 +4,6 @@ sylius_admin_partial_taxon_tree:
     defaults:
         _controller: sylius.controller.taxon:indexAction
         _sylius:
-            template: SyliusAdminBundle:Taxon:index.html.twig
+            template: @SyliusAdmin/Taxon/Partial/_tree.html.twig
```

Comment:
These all values with leading `@` should be inside quotation marks - http://symfony.com/blog/new-in-symfony-2-8-yaml-deprecations